test/integration/BasicTest.java

```diff
@@ -818,12 +818,12 @@ public void test_query_cancelled_asynchronously() throws IOException {
                         tx.close();
                         session.close();
                     }).start();
-                    tx.query().match(Graql.parseQuery("match $x isa thing;").asMatch());
+                    tx.query().match(Graql.parseQuery("match $x sub thing;").asMatch());
                 }).start();
             }
             Grakn.Session session = grakn.session(database, Arguments.Session.Type.DATA);
             Grakn.Transaction tx = session.transaction(Arguments.Transaction.Type.WRITE);
-            tx.query().match(Graql.parseQuery("match $x isa thing;").asMatch());
+            tx.query().match(Graql.parseQuery("match $x sub thing;").asMatch());
```

Comment:
Why do we need to change this, @flyingsilverfin ?

Reply:
errr I can move this to my other PR, however in the other PR type resolver complains this query is unsatisfiable - which is correct, there are no concrete subtypes of `thing`.

In the other PR i've made the "no abstract types are allowed on Thing variables" smarter than before, which is why it picks this up now. However probably makes sense to move this change over there!

Reply:
removed from this PR

Reply:
yes it does make sense to keep that change scoped in the right PR, @flyingsilverfin